.travis.yml

```diff
@@ -28,7 +28,7 @@ script:
 - yarn run build
 - if [ "x$TRAVIS_TAG" != "x" ]; then yarn run minimize ; fi
 - yarn run unitTests
-- if [ "x$TRAVIS_TAG" != "x" ]; then yarn run accessibilityTests ; fi
+- yarn run accessibilityTests
```

Comment:
What's the reason for running the accessibility tests on commits without tags? Wouldn't this lengthen the pipeline time considerably?

Reply:
This was only added temporarily and isn't intended to be merged. It's for testing on this PR specifically.